rpm/qtdeclarative.spec

```diff
@@ -112,6 +112,7 @@ Summary:    Qt Declarative QtQmlDevTools - development files
 Requires:   %{name} = %{version}-%{release}
 Requires:   %{name}-devel = %{version}-%{release}
 Requires:   qt5-qtdeclarative-devel = %{version}-%{release}
+Requires:   qtchooser
```

Comment:
qtdeclarativetools-devel? This doesn't look like the right subpackage.

Reply:
Is it devel-tools instead?